modules/apps/callbacks/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM golang:1.20-alpine3.18 as builder
```

Comment:
Leaving this comment as reminder in case https://github.com/cosmos/ibc-go/pull/4398 gets merged first

Reply:
```suggestion
FROM golang:1.21-alpine3.18 as builder
```